projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdfActivity.java

```diff
@@ -32,179 +39,243 @@
   // TODO: Add logs.
   private static final String TAG = "LawsuitPdfActivity";
   private final int STORAGE_PERMISSION_CODE = 1;
-
-  // filenames
-  private static final String LAWSUIT_MAIN_DIR_NAME = "lawsuits";
-  private static final String LAWSUIT_OUTPUT_DIR_NAME = "pdf";
-
-  // absPaths
-  private String lawsuitMainPath = "";
-  private String lawsuitOutputPath = "";
+  private boolean permissionGranted = false;
 
   // Formats
   private static final String TIME_ZONE = "Asia/Jerusalem";
```

Comment:
If we want to keep the consistency of the code, I think this suppose to be in the String.xml file, same for moreThanFiveLawsuits, lessThanFiveLawsuits, claimCaseHaser, etc

Reply:
This, what you are mentioning was actually in a past "TODO" for storing these Strings as resources in one of my previous PR. But I advised from @oferb to leave them as they are so I removed the TODO.

Perhaps now better off to move them to res?